pkg/console/subresource/deployment/deployment.go

```diff
@@ -321,6 +338,44 @@ func consoleContainer(cr *operatorv1.Console, volConfigList []volumeConfig, prox
 	}
 }
 
+func redirectContainer(redirectURL string) corev1.Container {
+	command := []string{
+		"console",
+		"redirect",
+		fmt.Sprintf("--redirect-url %s", redirectURL),
+	}
+
+	return corev1.Container{
```

Comment:
I don't think we need a separate container or new command. Couldn't the operator or console backend just listen on a different port for redirects?

Reply:
We have been talking with @benjaminapetersen regarding the approach and we agreed that it would be best to have all this custom logic been handled by operator, instead of putting it into the console server. Not really sure if the redirect logic/code belongs there. On the other hand operator should be the right place since it should handle all customisations.

Reply:
But why does it take a separate container and command? Why not just expose a port from the existing process?

Reply:
It feels a little odd to put the redirect in the operator imo. The operator should be responsible for deploying and maintaining the app. A redirect to the correct URL feels like a function of the app itself, which would be the console backend's responsibility.

Reply:
But having a server that should get a hostname from which it should redirect to the custom hostname and then watching two different hostnames seems weird.

Reply:
Console backend only needs to know the its current public URL, which it already has. It doesn't care where it's redirecting from. That is managed by the operator when it creates the service and updates the default route.